TimeReport.py

```diff
@@ -200,8 +206,13 @@ def outputToExcel(ws, dayOfWeek, fteTime, contTime, ignoreList):
 	parser = argparse.ArgumentParser(description="Calculate time reports.")
 	parser.add_argument("--date", help="mm.dd.yyyy of date to gather")
 	parser.add_argument("--ignore", help="comma seperated list of people to ignore")
+	parser.add_argument("--interns", help="Use JSON format as arguments for the intern's schedule. Example: {\"Aljon Preza\": [0,1,2]}")
```

Comment:
Please look into using enums to allow the JSON to be formatted as 'M', 'T', 'W', etc. This helps greatly with readability.